server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.objectcomputing.checkins.services.kudos;
+
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.data.annotation.Query;
+import io.micronaut.data.jdbc.annotation.JdbcRepository;
+import io.micronaut.data.model.query.builder.sql.Dialect;
+import io.micronaut.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+@JdbcRepository(dialect = Dialect.POSTGRES)
+public interface KudosRepository extends CrudRepository<Kudos, UUID> {
+
+        @Query(value = "SELECT " +
+                        "id, " +
+                        "PGP_SYM_DECRYPT(cast(message as bytea), '${aes.key}') as message, " +
+                        "senderid, teamid, datecreated, dateapproved, publiclyVisible " +
+                        "FROM kudos " +
+                        "WHERE dateapproved IS NULL " +
+                        "AND publiclyVisible IS TRUE " +
+                        "ORDER BY datecreated DESC")
```

Comment:
Needs `nativeQuery = true` and can be a text block for ease of editing and reading

```suggestion
        @Query(value = """
                SELECT
                    id, 
                    PGP_SYM_DECRYPT(cast(message as bytea), '${aes.key}') as message, 
                    senderid, teamid, datecreated, dateapproved, publiclyVisible 
                FROM kudos 
                WHERE dateapproved IS NULL 
                    AND publiclyVisible = TRUE 
                ORDER BY datecreated DESC""", nativeQuery = true)
```

Same for the other queries